doc/conf.py

```diff
@@ -117,6 +117,7 @@
         ("Examples", "auto_examples/index"),
         ("GUI", "gui/index"),
         ("API", "api"),
+        ("Network File", "network_file_desc"),
```

Comment:
don't add it here. You can add a line in the Readme linking to this file. It should say something to the effect that hnn-core uses hdf5 for IO and that the description of the file structure can be found here and the API of the functions for the IO can be found here